lib/Doctrine/DBAL/Driver/OCI8/OCI8Connection.php

```diff
@@ -71,17 +73,23 @@ public function __construct($username, $password, $db, $charset = null, $session
      */
     public function getServerVersion()
     {
-        if (! preg_match('/\s+(\d+\.\d+\.\d+\.\d+\.\d+)\s+/', oci_server_version($this->dbh), $version)) {
+        $version = @oci_server_version($this->dbh);
```

Comment:
We should most likely avoid introducing `@` where not 100000% sure if required. Yes, the DB2 extension is horrible, but what can fail here?